scripts/scil_tractogram_convert_trk_to_hdf5.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Save connections as TRK to HDF5.
+
+This script is useful to convert a set of connections or bundles to a single
+HDF5 file. The HDF5 file will contain a group for each input file, with the
+streamlines stored in the specified space and origin.
+
```

Comment:
Maybe, something like: Most of our scripts using hdf5 files deal with any space and origin, so just choose your favorite option?

Reply:
All of my scripts for connectivity assume vox/corner (to avoid as much computation on 100k connections). 

So I added: (keep the default if you are going to use the connectivity scripts in scilpy).